aboutme.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="homestyle.css">
```

Comment:
If you're using the same style sheet for all your pages, might be clearer to just name it a generic style.css.